javaScript.js

```diff
@@ -150,6 +151,27 @@ form.addEventListener(
   },
 );
 
+const getInfo = () => {
+  const formData = JSON.parse(localStorage.getItem('formData'));
+
+  if (formData) {
+    document.getElementById('full-name').value = formData.fullname;
+    document.getElementById('email').value = formData.email;
+    document.getElementById('message').value = formData.message;
+  }
+};
+
+form.addEventListener('keyup', () => {
+  const formData = {
+    fullname: document.getElementById('full-name').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
+  };
+
```

Comment:
- [ ] Please note that on page reload I lose all information typed into the form, kindly check and make sure the data is saved on the local storage before submitting again 👍🏽

Reply:
@rotimiazeez, I understand it, but it is a not good feedback